src/Authenticator/CookieAuthenticator.php

```diff
@@ -216,15 +211,14 @@ public function clearIdentity(ServerRequestInterface $request, ResponseInterface
     protected function _createCookie($value): CookieInterface
     {
         $data = $this->getConfig('cookie');
+        $name = $data['name'];
+        unset($data['name']);
+        $options = $data;
```

Comment:
This is an unnecessary assignment. You can directly use `$data` below.

Reply:
Wow. You are that quick. Thanks for your patience. I'm learning day-by-day.